tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "moduleResolution": "node",
     "strict": true,
     "baseUrl": ".",
-    "typeRoots": ["./src/@types", "./node_modules/@types"],
+    "typeRoots": ["./src/@types", "./src/shared/@types", "./node_modules/@types"],
```

Comment:
can remove src/@types

Reply:
Unfortunately, this solution still does not work.

In open-attestation-cli's package.json,
```
"@govtechsg/open-attestation": "github:eugbyte/open-attestation#experiment/types",
```

Will have to resolve it by creating our own documentLoader. as you suggested